backend/src/graphql/resolvers/mutations.js

```diff
@@ -0,0 +1,24 @@
+const bcrypt = require('bcrypt');
+const User = require('../../models/user');
+
+module.exports = {
+  register: async (_, { user }) => {
```

Comment:
Someone who freshly registers, can only create expenses right? If not, there needs to be an approve/reject mechanism